### ISSUE TITLE
Error when calling `SkipAttributeTransform` before decoding with npm module

### ISSUE THREAD

**Issue by ebogo1** [google/draco#742]
We ran into this issue with newer versions of draco3d in CesiumJS (I believe this started in 1.3.6). When we call `SkipAttributeTransform` before decoding, the decoder's `GetAttributeByUniqueId` returns a null pointer ~after the first attribute~ for attributes other than `NORMAL`, throwing an error when we try to destroy the empty attributeData: 

```
TypeError: Cannot read property '__destroy__' of undefined
    at Object.a.destroy (.../node_modules/draco3d/draco_decoder_nodejs.js:99:199)
```

To reproduce, unzip [draco-example.zip](https://github.com/google/draco/files/7220963/draco-example.zip) and run `npm install` then `node index.js`. If lines 67-70 are commented out, the decoder appears to run fine. The .glb in the folder was created with [gltf-pipeline](https://github.com/CesiumGS/gltf-pipeline) from the .gltf in the folder (originally from Khronos's [glTF-Sample-Models](https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/Box/glTF-Draco)).

**Comment by lilleyse** [comment]
As another data point, we only see this problem with the `draco.Mesh` geometry and not the `draco.PointCloud` geometry.

We had to temporarily disable quantization for glTF models in CesiumJS due to this issue and are eagerly waiting for a fix. It's always possible there's a mistake in our code so let us know if you spot anything unusual.

**Comment by ebogo1** [comment]
@ondys Thanks for the workaround fix - seems to work for us for most cases, we made the change in the PR above. Looking forward to the next Draco release!

**Comment by ondys** [comment]
This should be fixed in the current release 1.5.1

**Comment by lilleyse** [comment]
Thanks for the fix! We confirmed that this is now working in CesiumJS.
